File: exs/mundo_1/c++/017.cpp
/*

017 - Catetos e hipotenusa
  Faça um programa que leia o comprimento do cateto oposto (co) e do cateto
  adjacente (ca) de um triângulo retângulo, calcule e mostre o comprimento da
  hipotenusa

*/

#include <iostream>
#include <cmath>

using namespace std;

int main() {
  double cateto_oposto, cateto_adjacente, hipotenusa;

  cin >> cateto_oposto >> cateto_adjacente;
  hipotenusa = pow(pow(cateto_oposto, 2) + pow(cateto_adjacente, 2), 0.5);

  cout << "A Hipotenusa é igual a " << hipotenusa << "\n";
  return 0;
}

File: exs/mundo_1/java/017.java
// Faça um programa que leia o comprimento do cateto oposto (co) e do cateto
// adjacente (ca) de um triângulo retângulo, calcule e mostre o comprimento
// da hipotenusa

import java.util.Scanner;
import java.lang.*;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Cateto oposto: ");
        double co = scan.nextDouble();
        System.out.print("Cateto adjacente: ");
        double ca = scan.nextDouble();

        System.out.printf("Hipotenusa: %f\n", Math.hypot(co, ca));
    }
}
